static/app/components/events/contexts/chunk.tsx

```diff
@@ -100,6 +100,8 @@ class Chunk extends Component<Props, State> {
         return t('Trace Details');
       case 'otel':
         return t('OpenTelemetry');
+      case 'Memory Info':
```

Comment:
@mattjohnsonpint I noticed that the 'memory info' context always comes as "Memory Info" in the event. Do we think we can update it to be "memory_info" by default?

Reply:
We could in a future SDK release, sure.  But I would want to still recognized the old spelling also, so it works for existing versions.